docs/internal-api.md

```diff
@@ -96,7 +96,117 @@ module.exports = function() {
   });
 }
 ```
+You could get test stats when running with workers
```

Comment:
I like this but I think it is better to move this part of the documentation into 

https://github.com/codeceptjs/CodeceptJS/blob/3.x/docs/parallel.md

At least this is the place where user will look for it